tests/test_workflows.py

```diff
@@ -1043,11 +1043,10 @@ def test_cr_with_cm_operation():
 
 
```

Comment:
it should be great to complete tests with more (all ?) existing text extraction for errors. Maybe a comparison of text_extractions without taking into account spaces and carriage returns could be added for most of them

Reply:
A `set(text.splitlines())` comparison *may* be possible, but depending on your test PDFs, the existing `extract_text()` implementation is prone to including text in the wrong order (not just missing horizontal spacing). Test with [Epic Page.PDF](https://github.com/py-pdf/pypdf/files/13836944/Epic.Page.PDF) for an example.

Reply:
Still think this is a good item to pursue in the long run, but test cases covering all operators are now present.